packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -732,14 +742,143 @@ Link: <https://${yourOktaDomain}/api/v1/groups?after=00g1emaKYZTWRYYRRTSK&limit=
 ]
 ```
 
+#### List Groups with Search
+
+<ApiLifecycle access="ea" />
+
+Searches for groups based on the properties specified in the search parameter
+
+Property names in the search parameter are case sensitive, whereas operators (`eq`, `sw`, etc.) and string values are case insensitive.
+
+This operation:
+
+* Supports [pagination](/docs/reference/api-overview/#pagination).
+* Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding).
+For example, `search=type eq "OKTA_GROUP"` is encoded as `search=type+eq+%22OKTA_GROUP%22`.
+Examples use cURL-style escaping instead of URL encoding to make them easier to read.
+Use an ID lookup for records that you update to ensure your results contain the latest data. Search results are eventually consistent.
+* Searches many properties:
+   - Any group profile property, including imported app group profile properties.
+   - The top-level properties `id`, `created`, `lastMembershipUpdated`, `lastUpdated`, and `type`.
+   - The <ApiLifecycle access="ea" /> [source](#group-attributes) of groups with type of `APP_GROUP`, accessed as `source.id`.
+
+| Search Term Example                                       | Description                                                               |
+| :-------------------------------------------------------- | :------------------------------------------------------------------------ |
+| `type eq "APP_GROUP"`                                     | Groups that have a `type` of `APP_GROUP`                                  |
+| `lastMembershipUpdated gt "yyyy-MM-dd'T'HH:mm:ss.SSSZ"`   | Groups whose memberships were last updated after a specific timestamp     |
+| `id eq "00gak46y5hydV6NdM0g4"`                            | Groups with a specified `id`                                              |
+| `profile.name eq "West Coast Users"`                      | Groups that have a `name` of `West Coast Users`                           |
+| `profile.samAccountName sw "West Coast" `                 | Groups whose `samAccountName` starts with `West Coast`                    |
+| `source.id` eq `0oa2v0el0gP90aqjJ0g7`                     | Groups that have the source application with a specified `source.id`      |
+
+##### Search Examples
+
+List groups of type `APP_GROUP` that were created before `01/01/2014` and have a source application id `0oa2v0el0gP90aqjJ0g7`.
+
+    search=type eq "APP_GROUP" and (created lt "2014-01-01T00:00:00.000Z" and source.id eq "0oa2v0el0gP90aqjJ0g7")
+
+List groups that have a `name` that starts with `West Coast` or whose source application has the id `0oa2v0el0gP90aqjJ0g7` or have a `samAccountName` of `West Coast Users`.
```

Comment:
Hm, looking more carefully, there's a devious ambiguity here.  This can be read 2 ways:

1: List groups that ``have a `name` that starts with `West Coast` `` or ``whose source application has the id `0oa2v0el0gP90aqjJ0g7` `` or ``have a `samAccountName` of `West Coast Users`.``

2: List groups that ``have a `name` that starts with `West Coast` `` or whose source application (``has the id `0oa2v0el0gP90aqjJ0g7` or have a `samAccountName` of `West Coast Users` ``).

In other words, grouping the third clause as a sub-clause of `or whose source application` or as a subclause of `list groups that`.  The second interpretation is flat out wrong.  `samAccountName` has nothing to do with the source application

I'm not sure how to rewrite this to avoid the ambiguity, but I don't think we should leave it like this.

Reply:
hmm what about:
`List groups that have a `name` that starts with `West Coast` or have a `samAccountName` of `West Coast Users` or whose source application has the id `0oa2v0el0gP90aqjJ0g7``?